tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -659,6 +666,14 @@ namespace tuplex {
                            tstage->outputMode() == EndPointMode::MEMORY);
                     task->sinkOutputToMemory(outputSchema, tstage->outputDataSetID());
                 }
+
+                auto partitionId = uuidToString(partition->uuid());
+                auto info = tstage->partitionToExceptionsMap()[partitionId];
+                task->setNumInputExceptions(std::get<0>(info));
```

Comment:
perhaps using a struct instead of a tuple might be a good idea. Makes the meaning clearer...